app/build.gradle

```diff
@@ -201,7 +201,7 @@ dependencies {
     // name and the commit hash with the commit hash of the (pushed) commit you want to test
     // This works thanks to JitPack: https://jitpack.io/
     implementation 'com.github.TeamNewPipe:nanojson:1d9e1aea9049fc9f85e68b43ba39fe7be1c1f751'
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:v0.22.6'
+    implementation 'com.github.ChunkyProgrammer:NewPipeExtractor:c70bb83801ff4dee66822b3f89a3d412460b8727'
```

Comment:
This PR was opened before the extractor PR was merged, so this wasn't possible at this time. TobiGr will probably change this when they will rebase the PR, otherwise someone else will do it before it is merged.